pkg/pillar/iptables/ssh.go

```diff
@@ -103,6 +105,29 @@ func allowLocalPortRange(startPort int, endPort int) {
 		"-j", "REJECT", "--reject-with", "tcp-reset")
 }
 
+func allowUDPLocalPortRange(startPort int, endPort int) {
```

Comment:
Why not add a protocol string argument to allowLocalPortRange() instead of duplicating the function?

Reply:
For UDP we change the reject reason also to icmp destination unreachable. It is connection reset in case of TCP. Even if we pass the protocol to existing function, we will have to put if check and separate UDP, TCP cases.

Reply:
OK, but now we don't need this at all, do we?